### ISSUE TITLE
[Feature] Benchmarking Performance on General Devices

### ISSUE THREAD

**Issue by zhaochenyang20** [sgl-project/sglang#2488]
### Checklist

- [X] 1. If the issue you raised is not a feature but a question, please raise a discussion at https://github.com/sgl-project/sglang/discussions/new/choose Otherwise, it will be closed.
- [X] 2. Please use English, otherwise it will be closed.

### Motivation

We need to benchmark the speed performance of SGLang on various devices, at least different types of GPUs. This could give users a standard of the engine and whether their engines are working appropriately.

### Related resources

No such.

**Comment by bkutasi** [comment]
This is a good idea, but the codes move so fast, best would be a live bench (triggered on release) for smaller models and sometimes designated benches for large ones with different configs.

**Comment by zhaochenyang20** [comment]
> This is a good idea, but the codes move so fast, best would be a live bench (triggered on release) for smaller models and sometimes designated benches for large ones with different configs.

yeah. Sad that we do not have general devices, like A100 40G, L40, A800, A6000, 3090, 4090 etc
